pkg/ocm/ocm.go

```diff
@@ -38,6 +38,22 @@ func New(ocmConfigFile string) (ocmClient, error) {
 	return client, nil
 }
 
+// GetSupportRoleARN returns the support role ARN that allows the access to the cluster
+func (client ocmClient) GetSupportRoleARN(clusterID string) (string, error) {
```

Comment:
please test this

Reply:
@georgettica I think we should be fine without a unit test for this function since this really is no additional logic.

Reply:
I'll disagree, but ok

Reply:
What would a test look like? Its just a GET against an API and then returning an error if the field is blank, or the field. What logic is there to test?

Reply:
That is true, but I would like to cover all public APIs with a test, even if it doesn't make sense.

Just because of standard practices for all